pkg/controllers/binding/common.go

```diff
@@ -79,7 +81,17 @@ func ensureWork(c client.Client, workload *unstructured.Unstructured, overrideMa
 		return err
 	}
 
-	for _, targetCluster := range targetClusters {
+	var jobCompletions []workv1alpha2.TargetCluster
+	if workload.GetKind() == util.JobKind {
+		completions, ok, err := unstructured.NestedInt64(workload.Object, util.SpecField, util.CompletionsField)
```

Comment:
For [Parallel Jobs with a work queue](https://kubernetes.io/docs/concepts/workloads/controllers/job/#parallel-jobs), the `.spec.completions` usually is unset, and the `kube-apiserver` won't set this field even though it defaults to `.spec.parallelism`.

Reply:
Got it. So if the `.spec.completions` is not set, we should do nothing about the `.spec.completions` division.